.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: test
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8, 11, 17]
+        scala: [2.13.6]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+    - uses: coursier/cache-action@v6
+    - uses: actions/setup-java@v2
+      with:
+        distribution: adopt
```

```suggestion
    - uses: actions/setup-java@v2
      with:
        distribution: temurin
```

https://github.com/actions/setup-java/blob/v2.3.1/README.md says:

> **NOTE:** Adopt OpenJDK got moved to Eclipse Temurin and won't be updated anymore. It is highly recommended to migrate workflows from `adopt` to `temurin` to keep receiving software and security updates.